packages/lab/src/overlay/OverlayTrigger.tsx

```diff
@@ -0,0 +1,42 @@
+import { cloneElement, isValidElement, ReactNode, MouseEvent } from "react";
+import { mergeProps, useForkRef } from "@salt-ds/core";
+import { useOverlayContext } from "./OverlayContext";
+
+export interface OverlayTriggerProps {
+  children?: ReactNode;
```

Comment:
Is `children` optional intended?

Reply:
yup keeping it optional to account for cases where OverlayTrigger component is conditionally rendered

Reply:
so you could have an overlay rendered, but no overlay trigger?